_posts/2015-10-14-Four-days-DDD-course.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+name: When you get more than what you were expecting
+title: When you get more than what you were expecting
+date: 2015-10-14 09:35:00 +00:00
+author: Ana Nogal
+canonical:
+    name: my personal blog
+    href: http://www.ananogal.com/blog/when-you-get-more-than-what-you-were-expecting/
+image:
+    src: /assets/img/custom/blog/DDD_SkillsMatter.jpg
+tags:
+- DDD
+- Learning
+-
+---
+Almost 2 weeks ago, I attended a course at [SkillsMatter](http://skillsmatter.com/) about DDD. The first 2 days were intensive and a reinforce of a lot of concepts that I already had and learning a few more.
```

Comment:
The first 2 days were intensive and provided the opportunity to reinforce a lot of concepts I already knew as well as learn a few more.